src/shared/hooks/useStructuredData.ts

```diff
@@ -31,8 +33,10 @@ const useStructuredData = <Schema extends Thing>(
         try {
           content = JSON.stringify(value, undefined, 2);
         } catch {
-          // eslint-disable-next-line no-console
-          console.warn('Something went wrong when stringifying:', value);
+          logging.warn({
+            message: 'Something went wrong when stringifying',
+            data: value,
```

Comment:
This one is a bit of a problem, because there was an issue stringifying the data, then to log it, we'll have to stringify the data to send it to the remote logging systems 😅